### ISSUE TITLE
Can it be configured to work with a containerised esphome instance?

### ISSUE THREAD

**Issue by WizBangCrash** [esphome/esphome-vscode#20]
I would love to use this really useful extension, but I'm not sure how to configure it with my setup.

I have homeassistant and esphome running in separate docker containers on a NAS drive and have nothing running local on my PC.
I'm a little confused about the settings and whether I should be using `dashboard` or `local`, but setting either to use the URL of my esphome dashboard does not seem to work.
Are there ports on on the docker container I need to open up?

**Comment by screenagerbe** [comment]
Same question here. I'm using traefik with wildcard certificate in my docker setup., though.
The ESPhome dashboard is accessible on https://esphome.domain.local/
But I can't get VSCodium, which is running on a Win10 desktop in the same VLAN, to connect to the dashboard. I receive the error: `Cannot connect to ESPHome dashboard. Make sure you can access 'https://esphome.domain.local/' and have set the option 'leave_front_door_open': true `
So I can access the said url, but I have no idea about the setting leave_front_door_open': true

**Comment by glmnet** [comment]
> I have no idea about the setting leave_front_door_open': true

this is a configuration you find in the esphome add on settings. It basically disables security on the add on as there is no implementation to use credentials from this esphome vscode extension.

**Comment by screenagerbe** [comment]
> 
> 
> > I have no idea about the setting leave_front_door_open': true
> 
> this is a configuration you find in the esphome add on settings. It basically disables security on the add on as there is no implementation to use credentials from this esphome vscode extension.

Yeah, but I'm not running Home-Assistant OS, my environment is based on this install method: https://www.home-assistant.io/installation/raspberrypi#install-home-assistant-container
So no ESPHome add-on in my setup...
